_docs/vision.md

```diff
@@ -15,39 +15,39 @@ PostgreSQL?
 
 Welcome to Babelfish. The idea of the project is to close the gap between
 PostgreSQL and other systems by making PostgreSQL speak other protocols such as
-TDS, and maybe a lot more in the future. By implementing "MS SQL"-slang, we make
+TDS, and maybe a lot more in the future. By implementing "Microsoft SQL Server"-slang, we make
 it easier for people to move to PostgreSQL. 
 
 However, moving applications is just one side of the equation. Migrating data
 is one thing - making sure that applications work properly is another, far
 more important thing. In fact, migrating data to PostgreSQL can be relatively
```

Comment:
The first and second sentences refer to what is (at this point) non-existing content.  Maybe remove the first two sentences and lead with a more positive statement about migration (and make it active, singular, and remove 'enable'):

Migrating data to PostgreSQL can be relatively easy, while migrating an entire application stack can be difficult and time consuming. That's why Babelfish is so important. Babelfish allows you to take an existing application, connect it to a version of PostgreSQL that supports TDS, and run your application quickly and easily with comparatively minor adjustments.

Reply:
I feel fine with adding your new text, but I object to removing the current introduction.

After all, the name of the page is "vision". Should I rename the page to "current_status"??

Reply:
Personally, I would use 'Overview' or 'Introduction' instead of 'Vision'... probably 'Introduction'.  This page is wordy and wandering, and really doesn't get (clearly) to the point of why someone would use Babelfish without a lot of conjecture about someone else's technical abilities or the condition of their software.

Reply:
At this point, it may be easier if you push a commit with your ideas to this branch, as you are moving from corrections to wholesale rewrite.
Playing ping-pong like this wastes both our time, don't you think?

Reply:
> I feel fine with adding your new text, but I object to removing the current introduction.
> 
It still doesn't work for me - you're talking about two new subjects as if you've been talking about them already.  The word 'however' at the start of a sentence usually points to a contrast to a previous topic and the current topic.  You haven't talked about moving an application yet.  My proposed rewrite says everything that the first two sentences say.




> After all, the name of the page is "vision". Should I rename the page to "current_status"??

Reply:
> It still doesn't work for me - you're talking about two new subjects as if you've been talking about them already.

I don't get it.

> The word 'however' at the start of a sentence usually points to a contrast to a previous topic and the current topic. You haven't talked about moving an application yet.

Sure &ndash; in the sentence right before the "however".

Please push your suggestion to the branch, then I'll know what you mean.

Reply:
I'll wait until you check in your edits, and then check out the file and edit... I think it will be easier to follow the changes.

Reply:
The edits are "checked in" in the "doc-vidion-contributing" branch (I apologize for the typo).
I cannot merge them into the "main" branch until this pull request is approved.
So the options are to approve my changes, so that I can merge them, or to push to this branch.